INSTALL.md

```diff
@@ -42,7 +42,7 @@ curl -L https://git.io/n-install | bash -s -- -y lts
 
 ## Install node modules
 
-Make sure you're running the LTS version of NodeJS. Ignore any warnings.
+Make sure you're running the LTS version of NodeJS 8. Ignore any warnings.
```

Comment:
LTS will actually be bumped to 10 on october. I also had some issues with a older Node 8 version because the actual LTS started on 8.3. So NodeJS 8 LTS specifies that it should be NodeJS 8 after the 8.3 version.

Reply:
In that case, I think it would be clearer to write "NodeJS 8.3 or higher" throughout the document. We can update this when we migrate to Node 10.